packages/@react-spectrum/menu/src/context.ts

```diff
@@ -10,18 +10,21 @@
  * governing permissions and limitations under the License.
  */
 
-import {FocusStrategy} from '@react-types/shared';
-import {MenuTriggerState} from '@react-stately/menu';
+import {DOMProps, FocusStrategy, HoverEvents, PressEvents} from '@react-types/shared';
+import {MenuTreeState, MenuTriggerState} from '@react-stately/menu';
 import React, {HTMLAttributes, MutableRefObject, RefObject, useContext} from 'react';
 import {TreeState} from '@react-stately/tree';
 
-export interface MenuContextValue extends Omit<HTMLAttributes<HTMLElement>, 'autoFocus'> {
+export interface MenuContextValue extends Omit<HTMLAttributes<HTMLElement>, 'autoFocus' | 'onKeyDown'> {
   onClose?: () => void,
   closeOnSelect?: boolean,
   shouldFocusWrap?: boolean,
   autoFocus?: boolean | FocusStrategy,
   ref?: MutableRefObject<HTMLDivElement>,
-  state?: MenuTriggerState
+  state?: MenuTriggerState,
+  menuTreeState?: MenuTreeState,
+  onKeyDown?: (e: KeyboardEvent) => void,
+  onBackButtonPress?: () => void
```

Comment:
passed down through context just for MenuTrays so we can avoid adding it to the Menu props